### PR TITLE
add bloodhound-python

### LINKED ISSUE #2707: bloodhound python data collector
bloodhound python data collector
https://github.com/fox-it/BloodHound.py

### PR DESCRIPTION
**Pull request by noraj**
closes #2707

:warning: we must wait for impacket 0.9.21 (https://github.com/SecureAuthCorp/impacket/issues/799) to fix this issue https://github.com/fox-it/BloodHound.py/issues/41 before merging this :warning:

### CODE REVIEW

File: packages/bloodhound-python/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+package() {
+  cd "$_pkgname"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+}
```
Comment (Edu4rdSHL) [inline packages/bloodhound-python/PKGBUILD]: Please add final newline.

Comment (noraj) [reply packages/bloodhound-python/PKGBUILD]: This silly editor that remove it :/

Comment (Edu4rdSHL) [reply packages/bloodhound-python/PKGBUILD]: Yes, you need to add two newlines really. Thanks for the fix.

**Comment by noraj** [comment]
impacket 0.9.21 is here https://github.com/SecureAuthCorp/impacket/releases/tag/impacket_0_9_21
and is available in AL https://www.archlinux.org/packages/community/any/impacket/
we can merge this

**Pull request merged**